bitnami/thanos/Chart.yaml

```diff
@@ -15,4 +15,4 @@ maintainers:
 name: thanos
 sources:
   - https://github.com/bitnami/bitnami-docker-thanos
-version: 0.2.2
+version: 0.2.3
```

Comment:
I would consider this as a minor version, please, could you bump it to `0.3.0`?

Reply:
Done.